CHANGELOG.md

```diff
@@ -5,6 +5,7 @@
 - Default performance counters will be reported as metrics
 - When you instantiate `DependencyTrackingTelemetryModule` in code it will not detect certain http dependencies as Azure Storage calls. You need to register a telemetry initializer `HttpDependenciesParsingTelemetryInitializer` to enable this functionality. This telemetry initializer will be registered automatically during NuGet installation.
 - DependencyCollection nuget package was updated to Agent.Intercept nuget version XXX (TODO: update to the final version).
+- List of userAgent substrings that indicate that traffic is from a synthetic source was minimized for performance reasons. If you want to incude more substrings please add them under SyntheticUserAgentTelemetryInitializer/Filters. (List of filters that were used before is saved as a comment in the configuration file)
```

Comment:
I missed the one at the beginning which you just fixed, I was actually referring to the one on the end. ;)